kbase-extension/static/kbase/js/widgets/function_input/kbaseGrowthParamsPlotInput.js

```diff
@@ -14,7 +14,7 @@ define(['jquery',
         
         version: "1.0.0",
         options: {
-            loadingImage: "../images/ajax-loader.gif",
+            loadingImage: window.kbconfig.loading_gif,               
```

Comment:
Using window.kbconfig is deprecated, please start including narrativeConfig as a dependency, and using Config.get('loading_gif') for the spinner image. See here: https://github.com/kbase/narrative/blob/develop/docs/developer/widget_helper_modules.md#narrative-config for examples